src/Plugins/DBFTPlugin/Consensus/ConsensusService.OnMessage.cs

```diff
@@ -95,8 +95,11 @@ private void OnPrepareRequestReceived(ExtensiblePayload payload, PrepareRequest
             }
 
             // Timeout extension: prepare request has been received with success
-            // around 2*15/M=30.0/5 ~ 40% block time (for M=5)
-            ExtendTimerByFactor(2);
+            // around 4*3/M=12.0/5 ~ 80% block time (for M=5)
+            ExtendTimerByFactor(4);
```

Comment:
These coefficients is what I'm not so sure about, but if @vncoelho thinks it's better this way I'm not insisting on keeping them as is.

Reply:
In my tests here they behaved good.
And the adjusts were all empircally calculated with some technical knowledge..ahahha

Reply:
Just to mention, https://neospcc.medium.com/under-pressure-time-constrained-dbft-a4b4f30b73fd was done with the original ones. Worked fine as well. But that network had little delays overall, even at 100ms it was noticeably distant from the 25ms inter-DC latency.

Reply:
I think it is more needed when we have more delays, it is a protection as well. It does need change, at all, the time for the first round decision.
It just would alter round 2.